src/hassapi/client/base.py

```diff
@@ -17,15 +21,16 @@ class BaseClient(AuthenticatedClient):
     """Class for basic API client functionality."""
 
     def __init__(
-        self, hassurl: Optional[str] = None, token: Optional[str] = None, timeout: float = 3
+        self, hassurl: Optional[str] = None, token: Optional[str] = None, verify: Optional[bool] = None, timeout: float = 3

```

Comment:
booleans don't need to be optional since there are just 2 values
```suggestion
        self, hassurl: Optional[str] = None, token: Optional[str] = None, verify: bool = True, timeout: float = 3
```